src/maxdiffusion/loaders/lora_conversion_utils.py

```diff
@@ -0,0 +1,610 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
```

Comment:
HuggingFace copyright?

Reply:
maxdiffusion uses files from diffusers, this one being one of them. Per license, the copyright is kept with the original author.

Reply:
gotcha; i'm guessing this file is mostly just copy/paste then, so I'm not gonna read through all of it - but let me know if there's something in particular you wanted more eyes on :)